tests/run/worktree/main.fmf

```diff
@@ -0,0 +1,6 @@
+summary: Check whether worktree is correctly set for all steps.
+description:
+    For `prepare`, `finish` step and a simple test defined directly
+    in the execute step, the current working directory must be set
```

Comment:
For simple tests defined directly in the `execute` step we still use `discover/default/tests` as cwd.

Reply:
hmm, @psss so I guess that is something to fix? I do not think that is correct right?

Reply:
right, there is nothing to fix, just adjust the description.

Reply:
corrected test description,